src/freertos_drivers/ti/CC32xxSPI.hxx

```diff
@@ -206,7 +206,7 @@ private:
         do
         {
             /* fill TX FIFO but make sure we don't fill it to overflow */
-            if (tx_len && (rx_len - tx_len) < (8 * sizeof(T)))
+            if (tx_len && ((rx_len - tx_len) < (32 / sizeof(T))))
```

Comment:
The added parentheses should not be required, but also does no harm.

Reply:
there is actually a warning that some compiler versions produce, -Wsuggest-parentheses. So adding these proactively when convenient is a good idea.